vite.config.ts

```diff
@@ -6,8 +6,9 @@ function createURL(pathToFile: string) {
 
 export default defineConfig({
     test: {
-        include: ['./packages/**/tests/**/*.ts'],
+        include: ['./packages/**/tests/**/*.ts', 'packages/**/*.test.ts'],
```

Comment:
Please avoid adding this. This is part of the testing convention.

Reply:
We have applied this rule across the project. You can check other packages. They all have a tests/ in the root folder.

Reply:
the problem with `*.test.ts` is, you cannot split the test file and the non-test file into 2 different ts projects and gain the separate type check to benefit from project referencing.

Once you split test file into another `tests` folder in order to get project reference, the suffix `.test.ts` becomes unnecessary.